<launch>

    <arg name="color_topic" default="/camera/color/image_raw"/>
    <arg name="color_info" default="/camera/color/camera_info"/>

    <arg name="depth_topic" default="/camera/depth/image_rect_raw"/>
    <arg name="depth_info" default="/camera/depth/camera_info"/>

    

	<arg name="config_file" default="realtime_vdo.yaml"/>
    <rosparam command="load" file="$(find realtime_vdo_slam)/config/$(arg config_file)"/>

    <node pkg="realtime_vdo_slam" type="image_preprocessing_node_rgbd" name="vdo_preprocessing" output="screen">
        <param name="camera_mode" value="2"/>

        <param name="color_topic" value="$(arg color_topic)"/>
        <param name="color_info" value="$(arg color_info)"/>

        <param name="depth_topic" value="$(arg depth_topic)"/>
        <param name="depth_info" value="$(arg depth_info)"/>

    </node> 

</launch>